google/cloud/bigtable/benchmarks/benchmark.cc

```diff
@@ -149,7 +149,7 @@ void Benchmark::PrintThroughputResult(std::ostream& os,
   auto ops_throughput =
       1000 * result.operations.size() / result.elapsed.count();
   os << "# " << phase << " op throughput=" << ops_throughput << " ops/s"
-     << std::endl;
+     << "\n";
```

Comment:
In cases like this, it would be nice to include the "\n" within the previous string.

I might go about this by finding all lines that match `^\s*<<\sstd::endl`, then joining them with the previous line, then you could `s/\"\s+<<\s+std::endl/\n\"/`... or something like that :-)

Reply:
Yeah I've been screwing around with find/sed/grep for the past hour and can't seem to formulate a working regex for the join lines part. Feel free to take over. Closing this.